### DISCUSSION TITLE
Set whitelist for accessing /login.html

### DISCUSSION THREAD

**Discussion by Honglan233** [AdguardTeam/AdGuardHome#7166] [Q&A] [answered]
I want to set up an IP whitelist, only the IP in the whitelist can access /login.html

Do you have any good ideas? Thank you

**Comment by ainar-g** [comment, marked as answer]
You're better off running the web interface on e.g. `localhost` only and use the access control features of your reverse proxy (e.g. Nginx) or even your operating system (firewalls) to limit the access.
